kasa/tests/test_iotdevice.py

```diff
@@ -0,0 +1,259 @@
+"""Module for common iotdevice tests."""
```

Comment:
Shouldn't this go under an `iot` folder similar to the `smart` subfolder?

Reply:
Ohh, good catch, probably yes 😄